stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -379,7 +385,11 @@ internal class ChannelControllerImpl(
         val queryOnlineJob = if (domainImpl.isOnline()) { domainImpl.scope.async { runChannelQueryOnline(pagination) } } else { null }
         val localChannel = queryOfflineJob.await()
         if (localChannel != null) {
-            updateLiveDataFromLocalChannel(localChannel)
+            if (pagination.isLoadingMore) {
```

Comment:
Do we really need this extra property `isLoadingMore`?
Can't we do the same with the `pagination` property?

Reply:
I guess the this name is not good... I meant loading old messages, like "loading more" from the history. What about isLoadingFromHistory? We need this information because pagination is also used `loadOlderMessages` and `loadNewerMessages` and I need to differentiate then.

Reply:
`Pagination` is an enum and it is different depend on `loadOlderMessagess` (`Pagination.LESS_THAN`) and `loadNewerMessages` (`Pagination.GREATER_THAN`)
That is because I don't think we need this extra property

Reply:
Removed